package.json

```diff
@@ -17,12 +17,20 @@
     "build:remix": "remix build",
     "dev": "run-p dev:*",
     "dev:css": "npm run generate:css -- --dev",
-    "dev:remix": "remix dev",
+    "dev:remix": "PORT=8080 remix dev",
     "generate:css": "npx windicss './app/**/*.{jsx,tsx}' -to ./app/styles/windi.css -f windi.config.js",
-    "start": "remix-serve build",
-    "lint": "eslint --cache --cache-location ./node_modules/.cache/eslint .",
+    "start": "PORT=8080 remix-serve build",
```

Comment:
`PORT` is an environment variable. Not all runners use 8080 and this can break deployment.

Reply:
I will remove the port here